File: crates/deslop/tests/fixtures/fused-golden-csharp/shape_gamma.cs
namespace Golden.Shapes;

public sealed class TelemetryDescriptor
{
    public string Stream { get; } = "gamma-telemetry";
    public int Window { get; } = 47;
    public string Sink { get; } = "https://gamma.example.com/telemetry";

    public string Build(string device)
    {
        return Sink + "/" + device + "/" + Stream + "?window=" + Window;
    }
}

File: crates/deslop/tests/fixtures/fused-golden-typescript/shape_gamma.ts
export class TelemetryDescriptor {
  readonly stream = "gamma-telemetry";
  readonly window = 47;
  readonly sink = "https://gamma.example.com/telemetry";

  build(device: string): string {
    return this.sink + "/" + device + "/" + this.stream + "?window=" + String(this.window);
  }
}
